### PR TITLE
Update swagger for Gateway v1.0.2

### PR DESCRIPTION
**Pull request by wilwade**
# Purpose
The goal of this PR is 

Closes 

## Solution

A short description of what you have done to implement/fix the above mentioned feature or bug

@ any other developers who who worked on the PR with you

### Change summary
* A detailed list of bulleted
* changes that go into detail about
* the specifics of the changes
* to the codebase

## Steps to Verify
1. A numbered list of steps
2. To verify that this feature/bug
3. is now working

## Additional details / screenshot
- Any supplemental pictures or material
- Screenshot

### CODE REVIEW

File: backend/openapi-specs/account-service.json
Comment (wilwade) [inline backend/openapi-specs/account-service.json:1]: These are just copied over from Gateway `08b2fd4f62ad504ea6833a2628daa26c3018dcf8`

File: backend/services/AccountService.ts [hunk 1]
```diff
@@ -80,7 +80,7 @@ export class AccountService {
    */
   public async getAccount(msaId: string): Promise<Pick<AuthAccountResponse, 'msaId' | 'handle'>> {
     try {
-      const response = await this.client.AccountsControllerV1_getAccount(msaId);
+      const response = await this.client.AccountsControllerV1_getAccountForMsa(msaId);
```
Comment (wilwade) [inline backend/services/AccountService.ts:83]: Not sure how these worked before?

File: backend/services/ContentPublisherService.ts [hunk 2]
```diff
@@ -49,7 +49,7 @@ export class ContentPublisherService {
 
   public async postBroadcast(msaId: string, data: BroadcastDto): Promise<AnnouncementResponseDto> {
     try {
-      const res = await this.client.ContentController_broadcast(msaId, data);
+      const res = await this.client.ContentControllerV1_broadcast(msaId, data);
```
Comment (wilwade) [inline backend/services/ContentPublisherService.ts:52]: Not sure how these worked before?

File: backend/types/openapi-account-service.d.ts
Comment (wilwade) [inline backend/types/openapi-account-service.d.ts:1]: Generated

**Comment by JoeCap08055** [comment]
Superseded by Gateway 1.1

**Pull request closed without merging**